### ISSUE TITLE
NVDA version alpha-22117 cannot be installed properly

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#12219]
### Steps to reproduce:
First from [here](https://ci.appveyor.com/api/buildjobs/luhw4g6y56eca2cx/artifacts/output%2Fnvda_snapshot_alpha-22117%2C57e02cae.exe) download NVDA version alpha-22117
Then execute the installation process to reproduce the problem
### Actual behavior:
The installer cannot be executed properly
### Expected behavior:
Can be installed normally
#### NVDA installed/portable/running from source:
install version
#### NVDA version:
NVDA version alpha-22117
#### Windows version:
Microsoft Windows [version 10.0.19042.867]
#### Name and version of other software in use when reproducing the issue:
none
#### Other information about your system:
none
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
none
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Not in the scope of this tool
log:
[nvda.log](https://github.com/nvaccess/nvda/files/6202362/nvda.log)

**Comment by seanbudd** [comment]
Relevant traceback from the log provided

```
ERROR - RPC process 9024 (nvda_slave.exe) (13:47:49.574) - Dummy-4 (13264):
__main__.main:
Task failed, try again
Traceback (most recent call last):
  File "nvda_slave.pyw", line 64, in main
  File "installer.pyc", line 572, in install
  File "installer.pyc", line 530, in tryRemoveFile
installer.RetriableFailure: File C:\Program Files (x86)\NVDA\nvda.exe could not be removed
ERROR - gui.installerGui.doInstall (13:47:49.648) - MainThread (11624):
Failed to execute installer
Traceback (most recent call last):
  File "gui\installerGui.pyc", line 51, in doInstall
installer.RetriableFailure
```

AppVeyor page: https://ci.appveyor.com/project/NVAccess/nvda/builds/38393121/artifacts
PR that the alpha build references: https://github.com/nvaccess/nvda/pull/12215

**Comment by feerrenrut** [comment]
Is this issue still occurring?

**Comment by dingpengyu** [comment]
The current issue has been resolved and I will close this issue.
